packages/augur-tools/src/templates-template.ts

```diff
@@ -1178,3 +1323,9 @@ export const LIST_VALUES = {
     },
   ],
 };
+
+
+//##TEMPLATES##
+
+
+//##TEMPLATE_VALIDATIONS##
```

Comment:
?

Reply:
template string that gets replaced by generate-templates

Reply:
ohhhhh